src/gravel/controllers/services.py

```diff
@@ -110,6 +113,12 @@ def total_raw_reservation(self) -> int:
             total += (service.reservation * service.replicas)
         return total
 
+    @property
+    def available(self) -> int:
```

Comment:
This property isn't very clearly named. The object is `Services` so in the context of thinking about services "available" does not make me think of available space.